osmosis/src/setupStore.js

```diff
@@ -83,6 +83,7 @@ const setupStore = (useCustomHook, config = {}) => {
         <WrappedComponent {...props} />
       </StoreContextWrapper>
     );
+    Wrapper.displayName = `${useCustomHook.name}Wrapper`;
```

Comment:
Should there be a space between the curly brace and the word Wrapper?

Reply:
no, this is intentionally as shown above to ensure the display name does not contain spaces 😉